core/src/builder/companion/src/lib.rs

```diff
@@ -23,16 +23,17 @@ mod parse;
 /// In order to generate a companion builder use the macro as shown below.
 ///
 /// ```rust
+///
 /// #[fudge::companion]
 /// struct Local {
 /// 	#[fudge::parachain(2000)]
-/// 	pub centrifuge: ParachainBuilder<Block, RtApi>,
+/// 	pub centrifuge: ParachainBuilder<PBlock, PRtApi, PCidp, PDp, ()>,
```

Comment:
Updated these but the doc-tests are still failing... Not sure how we wanna tackle this. Maybe just point to the integration test file?